conda/recipes/librmm/meta.yaml

```diff
@@ -79,39 +79,6 @@ outputs:
         {% endif %}
         - fmt {{ fmt_version }}
         - spdlog {{ spdlog_version }}
-    test:
-      commands:
-        - test -f $PREFIX/include/rmm/logger.hpp
```

Comment:
Could we just test for directory existence instead?

```yaml
  test:
    commands:
      - test -d "${PREFIX}/include/rmm
```

Reply:
Applied in 116a721.